rothschild/Cargo.toml

```diff
@@ -17,8 +17,14 @@ kaspa-txscript.workspace = true
 kaspa-utils.workspace = true
 
 clap.workspace = true
+criterion.workspace = true
```

Comment:
must be dev-dependency

Reply:
Fixed in [9727d1d](https://github.com/kaspanet/rusty-kaspa/pull/371/commits/9727d1ddbc983e2a332e208843f0819691742bc4)